snapcraft/meta.py

```diff
@@ -1,6 +1,6 @@
 # -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
 #
-# Copyright (C) 2015-2016 Canonical Ltd
+# Copyright (C) 2016 Canonical Ltd
```

Comment:
because, not sure how it worked, but I moved `meta.py` to `meta_legacy.py` and created a new `meta.py` which only has 2016 things in it.